_posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md

```diff
@@ -0,0 +1,168 @@
+    ssl_certificate_file: /usr/share/data-prepper/config/my-certificate.crt
+    ssl_key_file: /usr/share/data-prepper/config/my-certificate.key
+    ssl_fingerprint_verification_only: true
+    authentication:
+        mutual_tls:
+```
+
+
+## Directory structure
+
+Before the release of Data Prepper 2.0, we distributed Data Prepper as a single executable JAR file. While convenient, 
+it made it difficult for us to include custom plugins.
+
+We now distribute Data Prepper 2.0 in a bundled directory structure. This structure features a shell script to launch 
+Data Prepper and dedicated subdirectories for JAR files, configurations, pipelines, logs, and more.
+
+```
+data-prepper-2.0.0/
+  bin/
+    data-prepper                    # Shell script to run Data Prepper
+  config/
+    data-prepper-config.yaml        # The Data Prepper configuration file
+    log4j.properties                # Logging configuration
+  pipelines/                             # New directory for pipelines
+    trace-analytics.yaml
+    log-ingest.yaml
+  lib/
+    data-prepper-core.jar
+    ... any other jar files
+  logs/
+```
+
+You now can launch Data Prepper by running `bin/data-prepper`; no need for additional command line arguments or Java system 
+property definitions. Instead, the application loads configurations from the `config/` subdirectory.
+
+Data Prepper 2.0 reads pipeline configurations from the `pipelines/` subdirectory. You can now define pipelines across 
+multiple YAML files in the subdirectory, where each file contains the definition for one or more pipelines. The directory 
+also helps keep pipeline definition distinct and, therefore, more compact and focused.
+
+## JSON & CSV parsing
+
+Many of our users have incoming data with embedded JSON or CSV fields. To help in these use cases, Data Prepper 2.0 
+supports parsing JSON or CSV.
+
+For example, when one large object includes a serialized JSON string, you can use the `parse_json` processor to extract 
+the fields from the JSON into your event.
+
+Data Prepper can now import CSV or TSV formatted files from Amazon S3 sources. This is useful for systems like Amazon CloudFront 
+which write their access logs as TSV files. Now you can parse these logs using Data Prepper. 
+
+Additionally, if your events have 
+CSV or TSV fields, Data Prepper 2.0 now contains a `csv` processor that can create fields from your incoming CSV data.
+
+## Other improvements
+
+Data Prepper 2.0 includes a number of other improvements. We want to highlight a few of them.
+
+* The OpenSearch sink now supports `create` actions for OpenSearch when writing documents. Pipeline authors can configure their pipelines to only create new documents and not update existing ones.
+* The HTTP source now supports loading TLS/SSL credentials from either Amazon S3 or Amazon Certificate Manager. Pipeline authors can now configure them for their log ingestion use cases. Before Data Prepper 2.0, only the OTel Trace Source supported these options.
```

Comment:
I believe either `SSL/TLS` or `TLS/SSL` is in use. I intentially chose `TLS/SSL` because we are using TLS. The SSL part is mostly there for historical reasons.

You can also see that the term `TLS/SSL` is used in the following Wikipedia article.

https://en.wikipedia.org/wiki/Transport_Layer_Security